### ISSUE TITLE
Windows build server failing to package

### ISSUE THREAD

**Issue by lordofhyphens** [alexrj/Slic3r#3633]
Appveyor build is failing to package Slic3r (sigh), although it is building.

Log: https://ci.appveyor.com/api/buildjobs/722jhin9c2cwn508/log

Snippet:
````
00:08:07] Files found in blib\arch: installing files in blib\lib into architecture dependent library tree
[00:08:07] Installing C:\projects\slic3r\LOCAL-~1\lib\perl5\MSWin32-x86-multi-thread-64int\auto\Slic3r\XS\XS.bs
[00:08:07] Installing C:\projects\slic3r\LOCAL-~1\lib\perl5\MSWin32-x86-multi-thread-64int\auto\Slic3r\XS\XS.xs.dll
[00:08:07] Installing C:\projects\slic3r\LOCAL-~1\lib\perl5\MSWin32-x86-multi-thread-64int\Slic3r\XS.pm
[00:08:07] OK
[00:08:07] Successfully installed Slic3r-XS-0.01
[00:08:07] 1 distribution installed
[00:08:07] Attempting to create directory C:/projects/slic3r/t/../local-lib
[00:08:07] 
[00:08:08] t\angles.t ..................... ok
[00:08:08] t\arcs.t ....................... ok
[00:08:09] t\avoid_crossing_perimeters.t .. ok
[00:08:10] t\bridges.t .................... ok
[00:08:10] t\clean_polylines.t ............ ok
[00:08:10] t\clipper.t .................... ok
[00:08:11] t\collinear.t .................. ok
[00:08:13] t\combineinfill.t .............. ok
[00:08:13] t\config.t ..................... ok
[00:08:14] t\cooling.t .................... ok
[00:08:17] t\custom_gcode.t ............... ok
[00:08:17] t\dynamic.t .................... skipped: variable-width paths are currently disabled
[00:08:42] t\fill.t ....................... ok
[00:08:46] t\flow.t ....................... ok
[00:08:46] t\gaps.t ....................... ok
[00:08:57] t\gcode.t ...................... ok
[00:08:57] t\geometry.t ................... ok
[00:09:02] t\layers.t ..................... ok
[00:09:02] t\loops.t ...................... skipped: temporarily disabled
[00:09:07] t\multi.t ...................... ok
[00:09:13] t\perimeters.t ................. ok
[00:09:13] t\polyclip.t ................... ok
[00:09:15] t\pressure.t ................... ok
[00:09:15] t\print.t ...................... ok
[00:09:30] t\retraction.t ................. ok
[00:09:35] t\shells.t ..................... ok
[00:09:38] t\skirt_brim.t ................. ok
[00:09:38] t\slice.t ...................... skipped: temporarily disabled
[00:09:48] t\support.t .................... ok
[00:09:49] t\svg.t ........................ ok
[00:09:49] t\thin.t ....................... ok
[00:09:50] t\threads.t .................... ok
[00:09:54] t\vibrationlimit.t ............. ok
[00:09:54] All tests successful.
[00:09:54] Files=33, Tests=520, 106 wallclock secs ( 0.13 usr +  0.01 sys =  0.14 CPU)
[00:09:54] Result: PASS
[00:09:54] If you also want to use the GUI you can now run `perl Build.PL --gui` to install the required modules.
[00:09:55] App::cpanminus is up to date. (1.7042)
[00:09:55] Class::Accessor is up to date. (0.34)
[00:09:56] Growl::GNTP is up to date. (0.21)
[00:09:56] LWP::UserAgent is up to date. (6.15)
[00:09:57] Net::Bonjour is up to date. (0.96)
[00:09:58] OpenGL is up to date. (0.70)
[00:09:58] Socket is up to date. (2.024)
[00:09:59] Win32::API is up to date. (0.84)
[00:09:59] Win32::TieRegistry is up to date. (0.30)
[00:10:00] Wx is up to date. (0.9928)
[00:10:00] Wx::GLCanvas is up to date. (0.09)
[00:10:00] Make this is run from the perl command window.
[00:10:00] Requires PAR.
[00:10:01] PAR::Packer is up to date. (1.035)
[00:10:01] pp : # Use of runtime loader module Module::Runtime detected.  Results of static scanning may be incomplete.
[00:10:01] At C:\projects\slic3r\utils\package_win32.ps1:21 char:1
[00:10:01] + pp `
[00:10:01] + ~~~~
[00:10:01]     + CategoryInfo          : NotSpecified: (# Use of runtim... be incomplete.:String) [], RemoteException
[00:10:01]     + FullyQualifiedErrorId : NativeCommandError
[00:10:01]  
[00:10:02] C:\Perl\site\bin\pp.bat: Cannot find module constant/defer.pm (specified with -M)
[00:10:02] 
[00:10:02] copy : Cannot find path 'C:\projects\slic3r\slic3r.par' because it does not exist.
[00:10:02] At C:\projects\slic3r\utils\package_win32.ps1:212 char:1
[00:10:02] + copy ..\slic3r.par "..\slic3r-${current_branch}-${APPVEYOR_BUILD_NUMB ...
[00:10:02] + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[00:10:02]     + CategoryInfo          : ObjectNotFound: (C:\projects\slic3r\slic3r.par:String) [Copy-Item], ItemNotFoundException
[00:10:02]     + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.CopyItemCommand
[00:10:02]  
[00:10:02] 1
````

Note: Leaving this here as a marker for the problem so I can close it when I get this figured out.

**Comment by lordofhyphens** [comment]
Near as I can tell, perl is using local::lib appropriately here. 

The reason it failed here is because appveyor has a 1hr limit and between bootstrapping and building Wx we are basically up on the edge of that already. The general solution I've come up with at this point is to cache the local-lib dir. Unfortunately this is going to take some arguing with the build server to get it to build and package -.-; 

https://ci.appveyor.com/project/lordofhyphens/slic3r/build/1.3.0-test_appveyor-294

**Comment by lordofhyphens** [comment]
Well it packages, but at least on my system trying to launch crashes. https://bintray.com/lordofhyphens/Slic3r/slic3r_dev/1.3.0-dev has some manually packaged builds (from my laptop). 

Would appreciate it if someone else picked up a copy from https://bintray.com/lordofhyphens/Slic3r/Slic3r_Branches/1.3.0-test_appveyor-303 and let me know if it worked for them.

**Comment by lordofhyphens** [comment]
Script is largely fixed via 16df4df8f87cb035528f4447a1a21ac79eb2e1af and a previous commit. I think the crashes on load is something inherent to the build server; looking into caching local-lib from a working manual build and re-using it for the build server builds. 

If that works then I get to close this issue :)
